strings/anagram/isanagram.go

```diff
@@ -0,0 +1,41 @@
+/*isanagram.go
+description: Checks if two given strings are anagram or not
+details: An anagram is a word or phrase that's formed by rearranging the letters of another word or phrase
+Reference: https://www.geeksforgeeks.org/pangram-checking/
+Author : Venkatasubramanian */
+
+package anagram
+
+import "strings"
+
+func cleanString(text string) []rune {
+	cleanText := strings.Join(strings.Fields(text), "") // Remove spaces
+	return []rune(cleanText)
+}
+
+func isanagram(text1 string, text2 string) bool {
+	cleanText1 := cleanString(text1)
+	cleanText2 := cleanString(text2)
+	if len(cleanText1) != len(cleanText2) {
+		return false
+	}
+	charMap := make(map[rune]int)
+
+	for i := 0; i < len(cleanText1); i++ {
```

Comment:
You could've just as easily used the `range` operator to look over the string runes. That would have prevented you from keeping track of `i`  over and over.

IMO, its much better to use range here as it makes it easier to understand what you're trying to achieve.